cmd/nvidia-dra-plugin/sharing.go

```diff
@@ -389,3 +404,36 @@ func (m *MpsControlDaemon) Stop(ctx context.Context) error {
 
 	return nil
 }
+
+// detactSupportTimeSliceByArch Determine whether the architecture series
+// supports setting time slices based on the gpu architecture.
+func detectSupportTimeSliceByArch(arch string) bool {
```

Comment:
As mentioned, we can rather key off ComputeCapability >= 7.0 as @klueska suggested.

Reply:
done